Pages/aboutUs.js

```diff
@@ -0,0 +1,17 @@
+const Page = require("./page");
+var data = require("../Data/data.js");
```

Comment:
I think there is no need for AboutUs, PrivacyAndPolicy and TermsAndConditions page objects, since these links are always present on every screen in application (they can be part of page.js or "Footer.js" inherited from page.js)

Reply:
I added footer.js, because more things are to come in this section on the page (footer), also, footer.js inherits from page.js methods.